### PR TITLE
Update Fortnite-API.com

### PR DESCRIPTION
**Pull request by MirayXS**
Fortnite-API.com domain has changed. There's still the Fortnite-API.com domain, wich is supported,
the docs are moved to https://dash.fortnite-api.com
this pull request updates this migration.

Also updates:
```
1. English Description
2. Logo Image
3. Thumbnail Image
4. Color
5. Tags (1 new tag)
```

- I also added myself as a contributor because of this pull request. Hope is ok.

# Screenshots
## `/` - Documentation Home Page

image

## `/account` - Account Page

image

## `/about` - About Page

image

### CODE REVIEW

File: websites/F/Fortnite-API/dist/metadata.json [hunk 1]
```diff
@@ -7,20 +7,28 @@
...
   "description": {
-    "en": "Fortnite-API.com is an unofficial API for Fortnite."
+    "en": "A powerful API containing data of all cosmetics, daily shop, news and more!"
   },
-  "url": "fortnite-api.com",
+  "url": [
+    "dash.fortnite-api.com",
+    "fortnite-api.com"
+  ],
   "version": "5.4.3",
```
Comment (Alanexei) [inline websites/F/Fortnite-API/dist/metadata.json:24]: Bump version

**Comment by MirayXS** [comment]
Bump ver to 6.0.0?

**Comment by doomlerd** [comment]
> 
> 
> Bump ver to 6.0.0?

5.5.0

File: websites/F/Fortnite-API/dist/metadata.json (hunk 1 above)
Comment (MirayXS) [reply websites/F/Fortnite-API/dist/metadata.json]: Bumped to 5.5.0

**Comment by doomlerd** [comment]
Update your fork

**Comment by MirayXS** [comment]
OK!
`Merge branch 'master' into Fortnite-API.com/FIX`

**Comment by MirayXS** [comment]
Updating my fork again.

**Comment by MirayXS** [comment]
Updating my fork again.

**Comment by MirayXS** [comment]
Proof of API endpoints working

# Screenshots
## `/v1/banners` - API request [ GET Banners ]

image

## `/v1/banners/colors` - API request [ GET Banner Colors ]

image

## `/v1/stats/br/v2` - API request [ GET BR Stats V2 ]

image

## `/v2/aes` - API request [ GET AES ]

image

## `/v2/shop/br` - API request [ GET BR Shop ]

image

## `/v2/shop/br/combined`  - API request [ GET BR Shop Combined ]

image

## `/v2/creatorcode` - API request [ GET Creator Code ]

image

## `/v2/creatorcode/search` - API request [ GET Creator Code Search ]

image

## `/v2/creatorcode/search/all` - API request [ GET Creator Code Search All ]

image

## `/v2/cosmetics/br` - API request [ GET Cosmetics List ]

image

## `/v2/cosmetics/br/new` - API request [ GET New Cosmetics ]

image

## `/v2/cosmetics/br/search` - API request [ GET Cosmetics Search ]

image

## `/v2/cosmetics/br/search/all` - API request [ GET Cosmetics Search All ]

image

## `/v2/cosmetics/br/search/ids` - API request [ GET Cosmetics Search by IDs ]

image

## `/v2/news` - API request [ GET News ]

image

## `/v2/news/br` - API request [ GET News BR ]

image

## `/v2/news/stw` - API request [ GET News STW ]

image

##  `/v2/news/creative` - API request [ GET News Creative ]

image

**Comment by MirayXS** [comment]
Updating my fork

**Comment by frozaum** [comment]
Hello, as of right now, we won't be able to review your pull request due to the temporarily locking of this repository that we announced in #1946, so we gladly request your patience until everything is sorted out. :)

**Comment by Bas950** [comment]
Unlocked (No longer on hold) - Please update your fork to get your Pull Request reviewed, if you don't do it within the next 3 days the Pull Request will be closed.

**Comment by MirayXS** [comment]
> just update your fork.

I’ve talked with Bas on discord. 

I’m currently on a vacation, and can only use my mobile phone. Once my vacations are finished, I’ll send another pull request. Sorry 😥

**Comment by doomlerd** [comment]
> I’m currently on a vacation, and can only use my mobile phone. Once my vacations are finished, I’ll send another pull request, with an updated fork. Sorry 😥

No worries, open another PR when you're back.

**Pull request closed without merging**